src/elements/content-preview/ContentPreview.js

```diff
@@ -751,13 +768,15 @@ class ContentPreview extends React.PureComponent<Props, State> {
     loadPreview = async (): Promise<void> => {
         const {
             annotatorState: { activeAnnotationId } = {},
+            boxAnnotations,
             enableThumbnailsSidebar,
             fileOptions,
             onAnnotatorEvent,
             onAnnotator,
             previewExperiences,
             showAnnotationsControls,
```

Comment:
I recall that multiple booleans, including this one, need to be `true` for the annotations controls to show up and work as expected. How do all of these annotations-related booleans work together? Do we need `useV4BoxAnnotations` at all or can we use `showAnnotationsControls` and check for the absence of `boxAnnotations` and lazy-load it if needed?

Reply:
Spoke with product about the required of a flag and we settled with no flag and if BoxAnnotations object is passed at all it will just assume v4 annotations and the rest of the existing flags will still work as intended.